Update cost recs docs for GA (#29006)
DataDog/documentation: content/en/cloud_cost_management/recommendations/_index.md

---

{{< callout url="#" btn_hidden="true" header="Join the Preview!" >}}

> Cloud Cost Recommendations is now GA with support for AWS, Azure, and Google Cloud, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>

Reviewer: ```suggestion
Cloud Cost Recommendations includes support for AWS, Azure, and Google Cloud, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>
```

Reviewer: We should remove the callout entirely and move any relevant bits of this content to under ## Prerequisites

1. Enable [resource collection][3] in the **Resource Collection** tab on the [AWS integration tile][4] or [Azure integration tile][8].
1. Install the [Datadog Agent][5] (required for over-provisioned resource recommendations).

> *Cloud Cost Recommendations now supports customers' non-USD billing currencies. 

Reviewer: ```suggestion
**Note**: Cloud Cost Recommendations supports customers' non-USD billing currencies. 
```

## Recommendation action-taking

> You can now easily act on recommendations to save money and optimize costs. Cloud Cost Recommendations support Jira, 1-click Workflow Automation, and Datadog Case Management. Unused EBS and GP2 EBS volume recommendations also support 1-click Workflow Automation. See the following details for each action-taking options:

Reviewer: ```suggestion
You can easily act on recommendations to save money and optimize costs. Cloud Cost Recommendations support Jira, 1-click Workflow Automation, and Datadog Case Management. Unused EBS and GP2 EBS volume recommendations also support 1-click Workflow Automation. See the following details for each action-taking options:
```

> You can now take action on recommendations as easily as possible to save money and optimize your costs. Cloud Cost Recommendations now supports Jira, 1-click Workflow Automation actions, and Datadog Case Management. Today, unused EBS volume and GP2 EBS volume recommendations support 1-click Workflow Automation actions and more will support soon. More details for each action-taking option are below:

Reviewer 2: ```suggestion
You can now easily act on recommendations to save money and optimize costs. Cloud Cost Recommendations support Jira, 1-click Workflow Automation, and Datadog Case Management. Unused EBS and GP2 EBS volume recommendations also support 1-click Workflow Automation. See the following details for each action-taking options:
```
We want to remove references to future supported features.

> - Jira: Jira issue creation is supported in both the recommendation side panel and "Active Recommendations" list. Users can click "Crea Jira issue" in the side panel or multi-select recommendations in the "Active Recommendations" list and create Jira issues from there. Jira issues are auto-tagged indicating they relate to a cost recommendation and include a link back to the recommendation referenced.

Reviewer 2: ```suggestion
- **Jira**: Jira issue creation is available in both the recommendation side panel and the "Active Recommendations" list. You can create a Jira issue by clicking "Create Jira issue" in the side panel or by selecting multiple recommendations in the "Active Recommendations" list. Created Jira issues are automatically tagged to indicate their connection to a cost recommendation and include a link back to the referenced recommendation.
```
Suggest to reword a bit for clarity

> - 1-click Workflow Automation actions: Actions are supported for a select set of recommendations today (and will expand to more soon) wherein a user can click an action button like "Delete EBS Volume" to quickly perform the recommended action directly from Cloud Cost Management.

Reviewer 2: ```suggestion
- **1-click Workflow Automation actions**: Actions are available for a limited set of recommendations, allowing users to quickly execute suggested actions, such as clicking "Delete EBS Volume," directly within Cloud Cost Management.
```

> - Datadog Case Management: Users can go to the recommendation side panel and click "Create Case" to generate a case to manage and take action on recommendations. 

Reviewer 2: ```suggestion
- **Datadog Case Management**: Navigate to the recommendation side panel and click "Create Case" to generate a case and take action on recommendations. 
```
